.circleci/config.yml

```diff
@@ -1,53 +1,65 @@
+---
+lint_and_unit: &lint_and_unit
+  - delivery
+  - danger
+  - lint-yaml
+  - lint-markdown
+
 version: 2.1
 
 orbs:
-  kitchen: sous-chefs/kitchen@1.0.0
+  kitchen: sous-chefs/kitchen@2
 
 workflows:
   kitchen:
     jobs:
+      # Lint and Unit Test
+      - kitchen/yamllint:
+          name: lint-yaml
+      - kitchen/mdlint:
+          name: lint-markdown
       - kitchen/danger:
           name: danger
           context: Danger
-      - kitchen/lint:
-          name: lint
+      - kitchen/delivery:
+          name: delivery
       - kitchen/dokken-single:
           name: server-debian-8
-          suite: server
-          platform: debian-8
-          requires: [danger, lint]
+          suite: server-debian-8
```

Comment:
Why specific, you could just do server and have it cover them all, I guess it's how granular you want it?

Reply:
Yeah - I was just updating based on what was there - don't know a ton about this repo so I didn't want to mess with the test setup if it was granular for a reason